Makefile

```diff
@@ -4,11 +4,19 @@ BUILDDIR=build
 RUBY_VERSION=$(shell cat .ruby-version | perl -ne "chomp and print")
 RUBY_TAR=ruby-$(RUBY_VERSION).tar.gz
 RUBY_SRC=$(BUILDDIR)/ruby-$(RUBY_VERSION)
-RUBY_URL=ftp://ftp.ruby-lang.org/pub/ruby/2.0/$(RUBY_TAR)
+RUBY_URL=http://files/ruby/$(RUBY_TAR)
```

Comment:
This doesn't make sense for a public project though... we should figure out why it's not resolving instead? Or we can do `RUBY_URL ?= ...`, where the `?=` allows us to override the env var from outside.